src/ScikitLearn/exotic.jl

```diff
@@ -0,0 +1,28 @@
+# | model                  | build  | fitted_params | report | metadata | tests 1 | tests 2 |
+# | ---------------------- | ------ | ------------- | ------ | -------- | ------- | ------- |
+# | DummyClassifier        | ✓      | ✓             | ✗      | ✓        |  ✓      | ✓       |
+# | DummyRegressor         | ✗      | ✗             | ✗      | ✗        |  ✗      | ✗       |
+# | IsotonicRegressor      | ✗      | ✗             | ✗      | ✗        |  ✗      | ✗       |
+# | KernelRidgeRegressor   | ✗      | ✗             | ✗      | ✗        |  ✗      | ✗       |
+# | BGMClassifier          | ✗      | ✗             | ✗      | ✗        |  ✗      | ✗       |
+# | GMClassifier           | ✗      | ✗             | ✗      | ✗        |  ✗      | ✗       |
+
+DummyClassifier_ = ((ScikitLearn.Skcore).pyimport("sklearn.dummy")).DummyClassifier
```

Comment:
Could you explain a bit what this line if for?

Reply:
These (and all other `model_`) get used in the macro, specifically this line which is in the `_skmodel_fit`  function in `MLJModels/src/ScikitLearn/ScikitLearn.jl`:

```julia
skmodel = $(Symbol(modelname, "_"))($((Expr(:kw, p, :(model.$p)) for p in params)...))
```

Basically the machinery is as follows:

* assign the python constructor to a name with an underscore : `Model_`
* declare a struct with name  `Model`
* automatically define a function `fit` which will internally  call `Model_(...)` which will in turn use the python backend.

Reply:
Ah, I see. Thanks!